sedex.md

```diff
@@ -1,16 +1,18 @@
 # Sedex Context
+Sedex Messages are not meant to be implemented by external services.

```

Comment:
Do we have a definition on here of what an external service is? Why would we say it's only for internal services? Is the distinction that helpful? The Context 'Sedex' makes it clear that it's about sedex and an "external service" might or might not be interested in it. Most likely not, but that's up to the "external service" to deceid.

Reply:
I close this conversation as this point has been discussed separately.